docs/release-notes.rst

```diff
@@ -132,6 +132,7 @@ Upgrade Notes
      - Notes
    * - 23.10.0
      - | - In NV-IPAM v0.1.1, the IP Pools configurations are read from IPPool CRs instead of using a ConfigMap. Existing ConfigMap configuration will be automatically migrated to IPPools CRs as part of the upgrade process.
+       | - Dropped Multus CNI support for versions older than v4.1.0.
```

Comment:
Is it on 23.10.0? or should it be on next 24.10.0?

Reply:
you're right. fixed!